.pre-commit-config.yaml

```diff
@@ -14,11 +14,13 @@ exclude: "ui\/.*py$"
 # List of super useful formatters.
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
+    rev: v3.2.0
```

Comment:
I'm not personally against updating this, I just wondered what drove you to update it?

Reply:
3.2.0 fixes an issue on Windows regarding the shebang pre-commit hook.